### ISSUE TITLE
Little question

### ISSUE THREAD

**Issue by TXH-mercury** [charlesCXK/SA-Gate#5]
Hi,
Thank you for your great work for us !
Have you used multigrid as in the code to get the 52.4mIOU on the NYUD dataset?

**Comment by charlesCXK** [comment]
Yes.

**Comment by TXH-mercury** [comment]
@charlesCXK  Thanks!  Noticing the relatively big performance difference between NYUDv2 Datasets and SUNRGBD datasets ,I have also encounter this kind of big performace gap on those two datasets , But some methods like ACnet shows no explict performance gap . Have you figured out some reasons?

**Comment by charlesCXK** [comment]
Hi, I think there are two possible reasons.
1. Maybe we didn't use the suitable training setting for SUNRGBD, such as lr schedule, etc.
2. SUNRGBD has many wrong labels (as shown in our paper), while the labels of NYUDv2 are clean enough.

Unfortunately, we haven't figure out how to improve the performance on SUNRGBD (including baseline), but you could check their code and maybe you could find some tricks we ignored before.

**Comment by TXH-mercury** [comment]
All rights , very helpful ,  I prepare to try tuning some training hyperparameters first. Thanks very much .
